command/staticcommands/battle.py

```diff
@@ -1,46 +1,62 @@
 import characters
 import random
 import sys
+import command
 
 class battle(object):
 
     def __init__(self, assetContainer):
         self.assetContainer = assetContainer
 
     def execute(self):
-        enemies = [characters.enemy("Weakling", random.randint(3, 10), random.randint(0, 5), 10),
-                   characters.enemy("Strongling", random.randint(10, 15), random.randint(5, 10), 25)]
-        enemy = random.choice(enemies)
+        #Enemy formatted with: Name, Attack, Speed, HP, Gold, XP
+        enemies = [characters.enemy("Weakling", random.randint(1,3), 5, random.randint(4, 8), random.randint(0, 5), 10),
+                   characters.enemy("Strongling", random.randint(2,5), random.randint(8, 12), random.randint(10, 15), random.randint(5, 10), 25)]
+        encounter = random.randint(0,10)
+        if encounter <= 7: #Choose your enemy
+            enemy = enemies[0]
+        else:
+            enemy = enemies[1]
         print "You've encountered a {0}. It has {1} hp.".format(enemy.name, enemy.hp)
-        while self.assetContainer.player.hp > 0 and enemy.hp > 0:
+        while self.assetContainer.player.hp > 0 and enemy.hp > 0: #While both players are breathing
             status = self.assetContainer.player.get_status()
             for line in status:
                 print line
             l = raw_input("What do you do? (Fight/Use/Run)")
             print ""
             choice = l.lower()
             if choice == ("fight"):
-                self.player_attack(enemy)
-                print ""
-                self.enemy_attack(enemy)
-                print ""
-            elif choice == ("use"):
-                pass #TODO ADD USE OF ITEMS OR AT LEAST POTIONS
+                if self.assetContainer.player.speed > enemy.speed: #If you're faster, you go first
+                    self.player_attack(enemy)
+                    print ""
+                    self.enemy_attack(enemy)
+                    print ""
+                else:
+                    self.enemy_attack(enemy)
+                    print ""
+                    self.player_attack(enemy)
+            elif choice == ("use"): #Uses items, and the enemy takes its turn
+                command.cmdfactory.cmdfactory.factory("inv", self.assetContainer)
```

Comment:
If you want he player to use an item at this point, call
`self.player.use_cons('potion)`

If you need it to be interactive (e.g. inventory menu), consider moving the menu logic to another place (ideally in character or inventory) and having both the `inv` command and this part of the `battle` command call that function instead. 

A command should not know about the command factory. If this is the case you've got a factory creating a command creating a factory creating a command.